pkg/kubectl/autoscale.go

```diff
@@ -20,10 +20,9 @@ import (
 	"fmt"
 	"strconv"
 
+	autoscaling "k8s.io/api/autoscaling/v1"
```

Comment:
autoscaling have 2 versions: v1 and v2alpha1.
What is guideline when picking an external version?
Stable one or latest one? You pick stable one here.